patches/minecraft/net/minecraft/item/ItemArmor.java.patch

```diff
@@ -1,14 +1,64 @@
 --- ../src-base/minecraft/net/minecraft/item/ItemArmor.java
 +++ ../src-work/minecraft/net/minecraft/item/ItemArmor.java
-@@ -235,6 +235,8 @@
-         private final int[] field_78049_g;
+@@ -43,7 +43,7 @@
+         }
+     };
+     public final EntityEquipmentSlot field_77881_a;
+-    public final int field_77879_b;
++    public final int armorAmount;
```

Comment:
If you want to fix a method/field name, please use the MCP-Bot or its issue tracker.